frontend/src/components/annotation/CheckboxInput.vue

```diff
@@ -1,6 +1,24 @@
 <template>
   <InputErrorDisplay :state="state" :msg-error="msgError" :msg-success="msgSuccess">
-    <b-form-checkbox-group v-model="inputVal" :name="config.name" :options="options" :state="state" :stacked="config.orientation === 'vertical'"></b-form-checkbox-group>
+
+    <b-form-checkbox-group 
+      v-model="inputVal"
+      :stacked="config.orientation=='vertical'"
+      >
+      
+      <b-form-checkbox 
+        :key="option.value" 
+        :value="option.value" 
+        :state="state"
+        v-for="option in options"
+        
+        >
+        {{ option.text }}
+        <b-icon-question-circle v-if="option.helptext != null" :id="option.value" class="annotation-help-prompt"></b-icon-question-circle>
```

Comment:
Is `option.value` on its own enough for the `id` - does Vue scope these IDs to the containing group?  I'm just thinking there's likely to be a clash if a project config has more than one set of radios with the same values (e.g. several different questions all using the same 1-5 scale).

Reply:
Good spot, the `id` attributes are global and they should be unique, so in this case the browser will actually complain about ids not being unique if you have multiple checkbox questions with the same 1-5 scale. And yes looks like it might break the tooltip text as the `target` requires an `id` and will clash if we have multiple checkboxes with the same `id`.

It should be enough to make the `id` something like `config.name + option.value` as you should never be capturing labels with identical names.

Reply:
Looks like there's a few packages we can use to automatically generate unique ids as well ([https://stackoverflow.com/questions/34950867/vue-js-how-to-set-a-unique-id-for-each-component-instance](https://stackoverflow.com/questions/34950867/vue-js-how-to-set-a-unique-id-for-each-component-instance)). 

It's something we might need to think about when we start allowing nested widgets/labels.

Reply:
Or simply

```
      <b-form-checkbox 
        :key="option.value" 
        :value="option.value" 
        :state="state"
        v-for="(option, idx) in options"
        :name="config.name"
        >
        {{ option.text }}
        <b-icon-question-circle v-if="option.helptext != null" :id="config.name + idx" class="annotation-help-prompt"></b-icon-question-circle>
        <b-tooltip v-if="option.helptext != null" :target="config.name + idx" :title="option.helptext"></b-tooltip>
      </b-form-checkbox>
```

to generate sequential IDs `mygroup0`, `mygroup1`, ...

Reply:
Yes sounds like a better solution just in case the value is blank or have characters that's not supported in the `id` field.